Hazelcast.Net/Hazelcast.Config/GroupConfig.cs

```diff
@@ -25,8 +25,14 @@ namespace Hazelcast.Config
     /// </remarks>
     public sealed class GroupConfig
     {
+        /// <summary>
+        /// default group password
```

Comment:
/// Default group password. (capitalizing the first word; there are a couple of lines like this in the following code comments.)